modules/platform/nuxeo-platform-document-routing/nuxeo-routing-core/src/main/java/org/nuxeo/ecm/platform/routing/core/impl/DocumentRoutingServiceImpl.java

```diff
@@ -1388,4 +1392,52 @@ public void invalidateRouteModelsCache() {
     public boolean isWorkflowModel(final DocumentRoute documentRoute) {
         return documentRoute.isValidated();
     }
+
+    /**
+     * @since 11.3
+     */
+    @Override
+    public boolean canCreateInstance(CoreSession session, List<String> documentIds, String workflowModelName) {
+
+        DocumentModelList docs = session.getDocuments(documentIds.stream().map(IdRef::new).toArray(DocumentRef[]::new));
```

Comment:
Move the definition of `docs` further down, just before its use.

Reply:
done